src/test/librbd/exclusive_lock/test_mock_PreAcquireRequest.cc

```diff
@@ -44,8 +44,6 @@ using ::testing::SetArgPointee;
 using ::testing::StrEq;
 using ::testing::WithArg;
 
-static const std::string TEST_COOKIE("auto 123");
```

Comment:
A similar variable seems to be unused in `src/test/librbd/exclusive_lock/test_mock_PostAcquireRequest.cc` and `src/test/librbd/exclusive_lock/test_mock_PreReleaseRequest.cc`.  Is there a reason to clean up only one instance?

Reply:
My initial intention was to make as mirror revise as possible. But yes, I would clean up all instances as you said, and then ask for your review again~